files/chef-marketplace-cookbooks/chef-marketplace/recipes/config.rb

```diff
@@ -0,0 +1,12 @@
+directory '/etc/chef-marketplace' do
+  owner 'root'
+  group 'root'
+  mode '0775'
+  action :create
+end
+
+if File.exist?('/etc/chef-marketplace/marketplace.rb')
+  Marketplace.from_file('/etc/chef-marketplace/marketplace.rb')
```

Comment:
I see the point, but none of our other omnibus packages do that.  As it's actually really just a shim around injecting attributes during the Chef run it isn't really all that important of a module.  I also have a `Marketplace::Config` class that is actually used in the omnibus-ctl commands that does entirely different things.  For consistency and not overlapping on the namespace of the config I'm going to leave it as is.  If it becomes a problem down the road we can rename or add a submodule.